src/main/webapp/profileScript.js

```diff
@@ -100,18 +100,80 @@ function loadRoutes() {
 
 function addRoute(newRoute) {
   let card = document.createElement("div");
+  card.id = "route" + newRoute.routeId;
   let container = document.createElement("div");
-  let routeDetails = document.createElement("div");
+
   let routeImg = document.createElement("img");
-  let routeRating = document.createElement("div");
+  routeImg.src = "pictures/praga-small.jpg";
+  routeImg.alt = "praga";
+
+  let routeDetails = addRouteDetails(newRoute);
+  let routeRating = createRatingElement(newRoute);
 
   card.classList.add("card");
   container.classList.add("container");
   routeDetails.classList.add("route-details");
   routeImg.classList.add("route-img");
   routeRating.classList.add("rating");
 
-  routeDetails.innerHTML = newRoute["routeName"];
+  card.appendChild(container);
+  container.appendChild(routeDetails);
+  container.appendChild(routeImg);
+  container.appendChild(routeRating);
+
+  if (newRoute["userAccess"] != "OWNER") {
+    document.getElementById("shared-routes").appendChild(card);
+  } else if (newRoute["isCompleted"]) {
+    document.getElementById("completed-routes").appendChild(card);
+  } else {
+    document.getElementById("future-routes").appendChild(card);
+  }
+}
+
+function addRouteDetails(newRoute) {
+  let routeDetails = document.createElement("div");
+
+  let routeName = document.createElement("p");
+  routeName.className = "route-name";
+  routeName.innerHTML = newRoute.routeName.bold();
+
+  let button1 = document.createElement("button");
```

Comment:
nit: button1, button2 and button3 are not a descriptive names. Please use more descriptive names for the buttons.